ocs_ci/deployment/baremetal.py

```diff
@@ -22,7 +36,471 @@ class BAREMETALUPI(Deployment):
+            )
+            ocp_install_template_path = os.path.join(
+                "ocp-deployment", ocp_install_template
+            )
+            install_config_str = _templating.render_template(
+                ocp_install_template_path, config.ENV_DATA
+            )
+            install_config_obj = yaml.safe_load(install_config_str)
+            install_config_obj['pullSecret'] = self.get_pull_secret()
+            install_config_obj['sshKey'] = self.get_ssh_key()
+            install_config_obj['metadata']['name'] = constants.BM_DEFAULT_CLUSTER_NAME
+            install_config_str = yaml.safe_dump(install_config_obj)
+            install_config = os.path.join(self.cluster_path, "install-config.yaml")
+            install_config_backup = os.path.join(self.cluster_path, "install-config.yaml.backup")
+            with open(install_config, "w") as f:
+                f.write(install_config_str)
+            with open(install_config_backup, "w") as f:
+                f.write(install_config_str)
+
+        def create_manifest(self):
+            """
+            Creates the Manifest files
+            """
+            logger.info("creating manifest files for the cluster")
+            run_cmd(
+                f"{self.installer} create manifests "
+                f"--dir {self.cluster_path} "
+            )
+
+        def create_ignitions(self):
+            """
+            Creates the ignition files
+            """
+            logger.info("creating ignition files for the cluster")
+            run_cmd(
+                f"{self.installer} create ignition-configs "
+                f"--dir {self.cluster_path} "
+            )
+
+        def configure_storage_for_image_registry(self, kubeconfig):
+            """
+            Configures storage for the image registry
+            """
+            logger.info("configuring storage for image registry")
+            patch = " '{\"spec\":{\"storage\":{\"emptyDir\":{}}}}' "
+            run_cmd(
+                f"oc --kubeconfig {kubeconfig} patch "
+                f"configs.imageregistry.operator.openshift.io "
+                f"cluster --type merge --patch {patch}"
+            )
+
+        def destroy(self, log_level=''):
+            """
+            Destroy OCP cluster specific to BM UPI
+            """
+            # Skip destroy cluster for BM UPI
+            pass
+
+
+def clean_disk():
+    """
+    Perform disk cleanup
+    """
+    lvm_to_clean = []
+    workers = get_typed_nodes(node_type='worker')
+    ocp_obj = ocp.OCP()
+    for worker in workers:
+        out = ocp_obj.exec_oc_debug_cmd(
+            node=worker.name, cmd_list=["lsblk -nd -e252,7 --output NAME --json"]
+        )
+        logger.info(out)
+        lsblk_output = json.loads(str(out))
+        lsblk_devices = lsblk_output['blockdevices']
+        for lsblk_device in lsblk_devices:
+            base_cmd = """pvs --config "devices{filter = [ 'a|/dev/%s.*|', 'r|.*|' ] }" --reportformat json""" \
```

Comment:
break lines with `()`. 
please apply in other places

Reply:
Done